app/views/welcome/using.html.erb

```diff
@@ -47,7 +49,7 @@ Dryden, P. N. (2012). These Are a Few of My Favorite Things: The Cultural Meanin
 <li><p><a href="http://projectblacklight.org/">Blacklight</a>, a Ruby on Rails engine that provides a customizable discovery interface; and</p></li>
 <li><p><a href="https://github.com/samvera/hydra-head/wiki">HydraHead</a>, a Ruby on Rails gem, or software library, that works with Active Fedora to provide CUD (create, update, and delete) functions for repository objects, along with other content management actions.</p></li>
 </ul>
-<p>ADRL includes <a href="https://github.com/samvera/curation_concerns">Curation Concerns</a>, a Ruby on Rails gem, that gives Samvera developers a common toolset to manage a wide range of complex digital objects. This gem provides a consistent way to manage various types of complex digital objects and their relationships such as manuscript collections, photos and printed ephemera, simple and complex audio and video recordings. Because the underlying relationships are described using the <a href="https://github.com/duraspace/pcdm/wiki">Portland Common Data Model (PCDM)</a>, these digital objects can be shared with non-Samvera systems using standard internet protocols for sharing Linked Data. The ADRL metadata model is expressed using the <a href="https://www.w3.org/TR/2014/REC-rdf11-concepts-20140225/">Resource Description Framework (RDF)</a>. Archival Resource Keys (ARKs), created using the California Digital Library’s <a href="http://ezid.cdlib.org">EZID service</a>, provide unique, persistent identifiers for repository objects.</p>
+<p>ADRL includes <a href="https://github.com/samvera/curation_concerns">Curation Concerns</a>, a Ruby on Rails gem, that gives Samvera developers a common toolset to manage a wide range of complex digital objects. This gem provides a consistent way to manage various types of complex digital objects and their relationships such as manuscript collections, photos and printed ephemera, simple and complex audio and video recordings. Because the underlying relationships are described using the <a href="https://github.com/duraspace/pcdm/wiki">Portland Common Data Model (PCDM)</a>, these digital objects can be shared with non-Samvera systems using standard internet protocols for sharing linked data. The ADRL metadata model is expressed using the <a href="https://www.w3.org/TR/2014/REC-rdf11-concepts-20140225/">Resource Description Framework (RDF)</a>. Archival Resource Keys (ARKs), created using the California Digital Library’s <a href="http://ezid.cdlib.org">EZID service</a>, provide unique, persistent identifiers for repository objects.</p>
```

Comment:
Did you make any change here, or was it just a whitespace change?

Reply:
@dunn  "Linked Data" was changed to "linked data"

Reply:
Ah, gotcha!